renderer/render-markdown.js

```diff
@@ -164,16 +165,24 @@ function fixCheckListStyles() {
   };
 }
 
-const renderer = remark()
-  .use(emojiToGemoji)
-  .use(gemojiToImages)
-  .use(fixHeadings)
-  .use(fixCheckListStyles)
-  .use(slug)
-  .use([hljs, html], {
-    sanitize: false,
-  });
-
-module.exports = function renderMarkdown(text, callback) {
-  renderer.process(text, callback);
+function frontmatter(fmts) {
+  if (!fmts.length) {
+    return [() => {}];
+  }
+
+  return [remarkFrontmatter, fmts];
+}
+
+module.exports = function renderMarkdown(text, opts, callback) {
+  remark()
+    .use(emojiToGemoji)
+    .use(gemojiToImages)
+    .use(fixHeadings)
+    .use(fixCheckListStyles)
+    .use(slug)
+    .use(...frontmatter(opts.ignorefrontmatter))
```

Comment:
Could you perhaps use `Object.assign()` instead of the splat? Thanks!

Reply:
I don't think it can be polyfilled with just `Object.assign` (it uses `apply` under the hood) in this case. I will just refactor it not to use splat at all.

Reply:
Done